ompi/mca/coll/tuned/coll_tuned_decision_fixed.c

```diff
@@ -34,6 +36,14 @@
 #include "ompi/op/op.h"
 #include "coll_tuned.h"
 
+/*
+ * The new default fixed decision functions were generated based off of
+ * results that were gathered using the ompi-collectives-tuning package.
+ * These results were submitted by multiple OMPI developers on their clusters
+ * and were subsequently averaged to generate the algorithm switch points
```

Comment:
Is it possible to provide more information of the tunings? ( e.g. the node count, instance type, provider etc.), which will make it clear that the tunings can reflect the performance of common user cases?

Reply:
I'd prefer it remain relatively anonymous, the systems the tests were run on differ greatly and belong to other companies. The averaged information is all available in the google drive which is described in the commit description.

Reply:
OK I see, then this PR looks good to me.